scene/resources/curve.h

```diff
@@ -272,6 +285,14 @@ class Curve3D : public Resource {
 	static void _bind_methods();
 
 public:
+	enum RotationMode {
+		ROTATION_NONE,
+		ROTATION_Y,
+		ROTATION_XY,
+		ROTATION_XYZ,
+		ROTATION_ORIENTED
+	};
```

Comment:
It may be better to specify the Target Vector and Up Vector like LookAt.

Reply:
I agree. But maybe it is better to do it in follow up PRs, as that breaks API.